lib/utils/docker-js.js

```diff
@@ -112,28 +112,44 @@ const generateConnectOpts = function (opts) {
 
 	return Bluebird.try(function () {
 		const connectOpts = {};
-		// Firsly need to decide between a local docker socket
-		// and a host available over a host:port combo
+
+		// Start with docker-modem defaults which take several env vars into account,
+		// including DOCKER_HOST, DOCKER_TLS_VERIFY, DOCKER_CERT_PATH, SSH_AUTH_SOCK
+		// https://github.com/apocas/docker-modem/blob/v2.1.3/lib/modem.js#L15-L65
+		const Modem = require('docker-modem');
+		const defaultOpts = new Modem();
```

Comment:
Previously, this code would only run if neither `--docker` nor `--dockerHost` were provided in the command line, in which case `docker-modem` was used to process the DOCKER_HOST env var (https://github.com/apocas/docker-modem/blob/v2.1.3/lib/modem.js#L15-L65)
In order to take advantage of `docker-modem`'s parsing of additional env vars like DOCKER_CERT_PATH, I've moved the code here so it always runs, and then any options explicitly provided in the command line simply override any values captured from env vars.